com.github.alexkdeveloper.desktop-files-creator.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.github.alexkdeveloper.desktop-files-creator",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "desktop-files-creator",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
Why does the app need access to the whole home directory?
If it needs to write files only to `~/.local/share/applications`, add `--filesystem=xdg-data/applications` instead.
If it always use file chooser dialog to open/save files, then it probably shouldn't need static filesystem permissions at all.

Reply:
Hi! I just tried to configure the permissions as you showed (--filesystem=xdg-data/applications) and the program does not see the files. Chooser is empty. The user will not be able to specify the files.

Reply:
If the chooser is empty, given it's a GTK3 app, my guess is that you're using `FileChooserDialog` instead of `FileChooserNative`. The former doesn't support portals. Unless you add custom widgets or otherwise extend the dialogs, consider switching your app to use `FileChooserNative`.

Reply:
I use image preview in chooser. It was not possible to add it in Native Chooser. Is it possible to add the following options :ro to existing permissions? That is, to allocate the application directory for writing, and everything else is read-only?

Reply:
Sure, you can replace `home` with `home:ro` + `xdg-data/application`

Reply:
It's done! Updated the file.